microsoft-edge/devtools-guide-chromium/css/reference.md

```diff
@@ -257,12 +257,12 @@ While editing the value of a declaration, you can use the following keyboard sho
 
```

Comment:
above, change:

See [Change declaration values with keyboard shortcuts](#change-declaration-values-with-keyboard-shortcuts) for shortcuts for quickly incrementing or decrementing a value by 0.1, 1, 10, or 100 units.
to:
For shortcuts for incrementing or decrementing values, see [Change declaration values with keyboard shortcuts](#change-declaration-values-with-keyboard-shortcuts) below.

Change from:
Change declaration values with keyboard shortcuts
to:
Increment declaration values with keyboard shortcuts
b/c we already have findable heading text "Change decl value", above.

gh rendered: 
See [Change declaration values with keyboard shortcuts](#change-declaration-values-with-keyboard-shortcuts)